src/Components/test/E2ETest/ServerExecutionTests/InteropReliabilityTests.cs

```diff
@@ -0,0 +1,349 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Ignitor;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.E2ETesting;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Components.E2ETests.ServerExecutionTests
+{
+    public class InteropReliabilityTests : ServerTestBase<AspNetSiteServerFixture>
+    {
+        private const int DefaultLatencyTimeout = 500;
+
+        public InteropReliabilityTests(
+            BrowserFixture browserFixture,
+            AspNetSiteServerFixture serverFixture,
+            ITestOutputHelper output)
+            : base(browserFixture, serverFixture, output)
+        {
+            serverFixture.BuildWebHostMethod = TestServer.Program.BuildWebHost;
+        }
+
+        public BlazorClient Client { get; set; } = new BlazorClient();
```

Comment:
Is there a really good reason to use ignitor for this kind of testing VS selenium? Just thinking about it because I don't want us to wiggle too much on the testing strategy unless we're getting something out of it.

Reply:
Yes, I'm sending invalid payloads and testing the low level responses from the server.

Reply:
Things I can't do with our well behaving client.

Reply:
Ok dope! A comment might be useful